File: FileParserServer/entity.py
class Entity:
    e_id = 0
    tag_list = {'Nh': "人名", 'Ni': "机构名", 'Ns': '地名'}

    def __init__(self, _name="", _tag="", p_id=0):
        self.id = Entity.e_id
        Entity.e_id += 1

        self.name = _name
        self.category = ""
        self.tag = ""
        self.p_id = 0
        self.property = "{}"
        self.tag = _tag
        self.p_id = p_id
        self.property = ""

    def __str__(self):
        return "id: " + str(self.id) + ", name: " + self.name + ", category: " + self.category + ", tag: " + self.tag

    def to_dic(self):
        dic = {"eid": self.id, "pid": self.p_id, "name": self.name, "type": "Character",
               "property": self.property}
        return dic

File: SpringProject/src/main/java/com/example/demo/po/Entity.java
package com.example.demo.po;

import lombok.Data;

/**
 * @Author: Owen
 * @Date: 2021/3/7 14:20
 * @Description:
 */
@Data
public class Entity {

    private String eid;

    private Integer pid;

    private String name;

    private String type;

    private String property;

    public Entity(){}

    public Entity(String eid, String name, String type, String property){
        this.eid = eid;
        this.name = name;
        this.type = type;
        this.property = property;
    }

    public Entity(Integer pid, String eid, String name, String type, String property){
        this.eid = eid;
        this.pid = pid;
        this.name = name;
        this.type = type;
        this.property = property;
    }

    public String getEid() {
        return eid;
    }

    public void setEid(String eid) {
        this.eid = eid;
    }

    public Integer getPid() {
        return pid;
    }

    public void setPid(Integer pid) {
        this.pid = pid;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getProperty() {
        return property;
    }

    public void setProperty(String property) {
        this.property = property;
    }
}
